dev-packages/browser-integration-tests/suites/integrations/httpclient/httpClientIntegration/init.js

```diff
@@ -1,11 +0,0 @@
-import * as Sentry from '@sentry/browser';
```

Comment:
Removed this test because it is not needed anymore, we only have the `httpClientIntegration` syntax by now.